src/app_service/service/contacts/async_tasks.nim

```diff
@@ -75,3 +75,44 @@ const asyncRequestContactInfoTask: Task = proc(argEncoded: string) {.gcsafe, nim
       "publicKey": arg.pubkey,
       "error": e.msg,
     })
+
+type
+  AsyncGetProfileShowcaseForContactTaskArg = ref object of QObjectTaskArg
+    pubkey: string
+    validate: bool
+
+const asyncGetProfileShowcaseForContactTask: Task = proc(argEncoded: string) {.gcsafe, nimcall.} =
+  let arg = decode[AsyncGetProfileShowcaseForContactTaskArg](argEncoded)
+  try:
+    let response = status_go.getProfileShowcaseForContact(arg.pubkey, arg.validate)
+    arg.finish(%* {
+      "publicKey": arg.pubkey,
+      "validated": arg.validate,
+      "response": response,
+      "error": nil,
```

Comment:
Usually we use `"errror": ""`, so I'm not sure if `nil` can be handled the same

Reply:
fixed!